arch/x86/kvm/pkvm/mmu.c

```diff
@@ -255,3 +255,9 @@ int pkvm_host_mmu_map(unsigned long phys, unsigned long size, bool read,
 	/* The vaddr == phys for the host MMU */
```

Comment:
commit message: checkpatch warning:

```
WARNING: Possible repeated word: 'the'
#7: 
generic pkvm_pgtable_unmap(). Because the the host mmu is mapped with
```